src/pages/Portfolio/MyKiva/MyKivaPage.vue

```diff
@@ -255,18 +257,25 @@ const handleBackToJourney = () => {
 };
 
 const fetchLoanUpdates = loanId => {
-	apollo.query({ query: updatesQuery, variables: { loanId } })
+	apollo.query({ query: updatesQuery, variables: { loanId, limit: 3, offset: updatesOffset.value } })
 		.then(result => {
-			loanUpdates.value = result.data?.lend?.loan?.updates?.values ?? [];
 			totalUpdates.value = result.data?.lend?.loan?.updates?.totalCount ?? 0;
+			const updates = result.data?.lend?.loan?.updates?.values ?? [];
+			loanUpdates.value = loanUpdates.value.concat(updates);
 		}).catch(e => {
 			logReadQueryError(e, 'MyKivaPage updatesQuery');
 		});
 };
 
+const loadMoreUpdates = () => {
+	updatesOffset.value += 1;
```

Comment:
I guess this is what we're chatting about in slack. This probably needs to be 3, right?